app/helpers/metadata_helper.rb

```diff
@@ -19,15 +19,39 @@ def metadata_url
 
   def format_title(title)
     return "Exercism" unless title
-
     "#{title} | Exercism"
   end
 
   def metadata
     @metadata ||=
       case namespace_name
+      when "admin"
+        { title: "Admin" }
       when "my"
-        nil
+        case controller_name
+        when "notifications"
+          { title: "Notifications" }
+        when "settings", "track_settings", "preferences"
+          { title: "Settings" }
+        when "reactions"
+          { title: "My Reactions" }
+        when "solutions"
+          { title: "My #{@track.title}/#{@exercise.title}" }
```

Comment:
Could you change this to be `#{@track.title} | #{@exercise.title} (Solution)` pls?

Reply:
I left off "(Solution)" because this is the same page that gets rendered if they haven't submitted a solution yet.